.github/workflows/send-proofs-docker.yml

```diff
@@ -53,4 +53,4 @@ jobs:
 
     - name: Ensure admin permissions in _work
       if: always()
-      run: sudo chown admin:admin -R /home/admin/actions-runner/_work/
+      run: sudo chown admin:admin -R /home/admin/actions-runner/_work/
```

Comment:
fix no newline at the end of the file

(you can configure VIM to do this automatically)